### ISSUE TITLE
<19:02:51> Disconnected from server (parameter not found)

### ISSUE THREAD

**Issue by daffyyyy** [TeaSpeak/TeaSpeak#228]
## System Information
***Operating System***: debian9
***Operating System Type*** (x86|x64): x64
***TeaSpeak Version***:
1.1.39-beta
Packed version: 4
## Issue
***Reason for reporting this issue***:
Clients disconnecting without reason

***Detailed Description***:

```
[19:02:53] [INFO]   FILE | Got new client from IP:51527
[19:02:53] [DEBUG]  FILE | Got new file transfer via TS
[19:02:53] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_3881036020' (Type: download)
[19:02:53] [DEBUG]  FILE | File size. Having 6902 bytes. Max bytes 6902
[19:02:53] [DEBUG]  FILE | File transfer complete!
[19:02:53] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:02:55] [ERROR]     1 | Ticking of client IP:44041 (Klanowy RIP #2) needs more that 2500 microseconds! (2580 microseconds)
**[19:02:59] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:02:59] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:00] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:01] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:02] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!**
[19:03:03] [INFO]   FILE | Got new client from IP:51528
[19:03:03] [INFO]   FILE | Got new client from IP:51545
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_590151279' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 3361 bytes. Max bytes 3361
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_3556826734' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 718 bytes. Max bytes 718
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [INFO]   FILE | Got new client from IP:51522
[19:03:03] [INFO]   FILE | Got new client from IP:51534
[19:03:03] [INFO]   FILE | Got new client from IP:51535
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_3725390421' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 3531 bytes. Max bytes 3531
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_3414596882' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 3376 bytes. Max bytes 3376
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_3097187920' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 908 bytes. Max bytes 908
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [INFO]   FILE | Got new client from IP:51543
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/icons/icon_2220518364' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 644 bytes. Max bytes 644
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:03:03] [INFO]   FILE | Got new client from IP:51544
[19:03:03] [DEBUG]  FILE | Got new file transfer via TS
[19:03:03] [DEBUG]  FILE | Got file transfer for file './files/server_1/avatars/avatar_iadkceajkgflobmafbfmdkfigdglpfpkgfbbocpi' (Type: download)
[19:03:03] [DEBUG]  FILE | File size. Having 9674 bytes. Max bytes 9674
[19:03:03] [DEBUG]  FILE | File transfer complete!
[19:03:03] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
**[19:03:03] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:04] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!**
[19:03:05] [INFO]   FILE | Got new client from IP:51550
[19:03:05] [DEBUG]  FILE | Got new file transfer via TS
[19:03:05] [DEBUG]  FILE | Got file transfer for file './files/server_1/avatars/avatar_lcdkdokidifcejfgaggbpldgfeikdainhlhnhmhd' (Type: download)
[19:03:05] [DEBUG]  FILE | File size. Having 14273 bytes. Max bytes 14273
[19:03:05] [DEBUG]  FILE | File transfer complete!
[19:03:05] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
**[19:03:06] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:06] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:06] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:07] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:07] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:07] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:08] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:08] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:09] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!
[19:03:10] [DEBUG]     1 | Cant decrypt packet of type Command. Dropping packet. Error: memory verify failed!**
[19:03:12] [DEBUG]     1 | [IP:64689/Kube] Got a ping timeout. (Last successful ping: 21958ms ago)
[19:03:12] [DEBUG]     1 | [IP:64689/Kube] Closing voice client connection. (Flush: true)
[19:03:12] [DEBUG]     1 | [IP:57519/daffyy] Got a ping timeout. (Last successful ping: 21942ms ago)
[19:03:12] [DEBUG]     1 | [IP:57519/daffyy] Closing voice client connection. (Flush: true)
[19:03:12] [DEBUG]     1 | [IP:51138/DexteR] Got a ping timeout. (Last successful ping: 21916ms ago)
[19:03:12] [DEBUG]     1 | [IP:51138/DexteR] Closing voice client connection. (Flush: true)
[19:03:12] [DEBUG]     1 | [IP:51138/DexteR] Awaiting write prepare, write and acknowledge queue flushed
[19:03:12] [DEBUG]     1 | [IP:51138/DexteR] Write prepare queue progressed
[19:03:12] [DEBUG]     1 | [IP:51138/DexteR] Write and acknowledge queue are flushed
[19:03:12] [INFO]      1 | Voice client 92/v+lF9XzRxO8xuqPkXUlqs/QF2XA= (DexteR) from IP:51138 left.
[19:03:12] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_last_client_disconnect. New value: 1531587792. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:03:12] [DEBUG]     1 | [IP:57519/daffyy] Awaiting write prepare, write and acknowledge queue flushed
[19:03:12] [DEBUG]     1 | [IP:57519/daffyy] Write prepare queue progressed
[19:03:12] [DEBUG]     1 | [IP:64689/Kube] Awaiting write prepare, write and acknowledge queue flushed
[19:03:12] [DEBUG]     1 | [IP:64689/Kube] Write prepare queue progressed
[19:03:13] [DEBUG]     1 | [IP:64689/Kube] Cant flush io!
[19:03:13] [DEBUG]     1 | [IP:64689/Kube]   Acknowledge queue not empty!
[19:03:13] [INFO]      1 | Voice client 23/gDokCaZb4cBRXDpYY2v1+mUR4vg= (Kube) from IP:64689 left.
[19:03:13] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_last_client_disconnect. New value: 1531587793. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:03:13] [DEBUG]     1 | [IP:57519/daffyy] Cant flush io!
[19:03:13] [DEBUG]     1 | [IP:57519/daffyy]   Acknowledge queue not empty!
[19:03:13] [INFO]      1 | Voice client 4/dfuTNJShoKvYr36X6Rjj59IGA50= (daffyy) from IP:57519 left.
[19:03:16] [DEBUG]     1 | [IP:57519/undefined] Creating VoiceClient instance at 0x7f155f5884d0
[19:03:16] [DEBUG]     1 | Allocated new voice client connection at 0x7f155f4f31d0
[19:03:16] [DEBUG]     1 | Remote address (v4) IP destination address IP
[19:03:16] [DEBUG]     1 | Having new voice client. Remote address: IP:57519
[19:03:16] [DEBUG]     1 | [IP:57519/undefined] Got initivexpand acknowlage!
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_totalconnections' for 4 (New value: 57, SQL: UPDATE `clients` SET `connections` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [DEBUG]   GEN | Query permission manager
[19:03:16] [DEBUG]     1 | Loaded client from database. Database id: 4 Unique id: dfuTNJShoKvYr36X6Rjj59IGA50=
[19:03:16] [DEBUG]     1 | [IP:57519/daffyy] Got client init. (HWID: 8aee6cfeb259e8eab8f810609ff19655,e9e5c8cc978b4992c6a3e7bc616412d0)
[19:03:16] [DEBUG]     1 | [Permission] Value test result for test type 0.
[19:03:16] [DEBUG]     1 | [Permission]    Permission: b_client_ignore_bans Required value: 1 Gained value: 1 Force required: 1 Channel: none Result: 1
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_lastconnected' for 4 (New value: 1531587796, SQL: UPDATE `clients` SET `lastConnect` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_totalconnections' for 4 (New value: 58, SQL: UPDATE `clients` SET `connections` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [ERROR]     1 | Command handling of command clientinit needs 20ms
[19:03:16] [ERROR]     1 | [IP:57519/daffyy] Handling of packet Command needs more than 10ms (20ms)
[19:03:16] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_last_client_connect. New value: 1531587796. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:03:16] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_client_connections. New value: 625. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:03:16] [DEBUG]   GEN | Client daffyy|IP comes from Poland|PL
[19:03:16] [DEBUG]     1 | [IP:57519/daffyy] Got a channel talk power of 100 Talk power set is 0
[19:03:16] [DEBUG]     1 | Client id: 1
[19:03:16] [DEBUG]     1 | [IP:57519/daffyy] Got a channel talk power of 100 Talk power set is 0
[19:03:16] [DEBUG]     1 | [Property] Changed property in db key: client_nickname value: daffyy
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_nickname' for 4 (New value: daffyy, SQL: UPDATE `clients` SET `lastName` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [DEBUG]     1 | [Property] Changed property in db key: client_version value: 3.1.10 [Build: 1528537615]
[19:03:16] [DEBUG]     1 | [Property] Changed property in db key: client_platform value: Windows
[19:03:16] [DEBUG]     1 | [Property] Changed property in db key: hwid value: 8aee6cfeb259e8eab8f810609ff19655,e9e5c8cc978b4992c6a3e7bc616412d0
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_lastconnected' for 4 (New value: 1531587796, SQL: UPDATE `clients` SET `lastConnect` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [DEBUG]     1 | [Property] Changing client property 'client_totalconnections' for 4 (New value: 58, SQL: UPDATE `clients` SET `connections` = :value WHERE `serverId` = :serverId AND `cldbid` = :cldbid)
[19:03:16] [DEBUG]     1 | [Property] Changed property in db key: connection_client_ip value: IP
[19:03:16] [INFO]      1 | Voice client 4/dfuTNJShoKvYr36X6Rjj59IGA50= (daffyy) from IP:57519 joined.
[19:03:16] [DEBUG]     1 | [IP:51138/undefined] Creating VoiceClient instance at 0x7f155c4cc520
[19:03:16] [DEBUG]     1 | Allocated new voice client connection at 0x7f155c4ddb30
[19:03:16] [DEBUG]     1 | Remote address (v4) IP destination address IP
[19:03:16] [DEBUG]     1 | Having new voice client. Remote address: IP:51138
[19:03:16] [DEBUG]   GEN | [IP:57519/daffyy]Update manager variable via clientupdate. Variable: 'client_badges' value: 'overwolf=0:badges=534c9582-ab02-4267-aec6-2d94361daa2a,f22c22f1-8e2d-4d99-8de9-f352dc26ac5b'
[19:03:16] [DEBUG]     1 | [IP:51138/undefined] Got initivexpand acknowlage!

```

*Delete me: If you dont follow this issue template, you may not get a response or the issue get instandly closed.*

**Comment by WolverinDEV** [comment]
Does it directly happen when you join?
And with which client version are you joining?

**Comment by daffyyyy** [comment]
From my perspective, people talk and sometimes kick out of the server. I did not notice anyone coming in or out.
Nothing on the server is changed, as shown in the logs, downloading icons + conversation

**Comment by daffyyyy** [comment]
A few more:
```
[19:20:44] [DEBUG]   GEN | Invalid packet id pushback. Current index: 35 packet index: 34 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0000 connection_server2client_packetloss_keepalive=0.0076 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0002)
[19:20:45] [DEBUG]   GEN | Invalid packet id pushback. Current index: 35 packet index: 34 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0000 connection_server2client_packetloss_keepalive=0.0076 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0002)
[19:22:47] [DEBUG]   GEN | Invalid packet id pushback. Current index: 37 packet index: 36 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.1988 connection_server2client_packetloss_keepalive=0.0050 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.1909)
[19:22:51] [DEBUG]   GEN | Invalid packet id pushback. Current index: 37 packet index: 36 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.1988 connection_server2client_packetloss_keepalive=0.0050 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.1909)
[19:22:53] [INFO]   FILE | Got new client from IP:56249
[19:22:53] [DEBUG]  FILE | Got new file transfer via TS
[19:22:53] [DEBUG]  FILE | Got file transfer for file './files/server_1/avatars/avatar_lcdkdokidifcejfgaggbpldgfeikdainhlhnhmhd' (Type: download)
[19:22:53] [DEBUG]  FILE | File size. Having 14273 bytes. Max bytes 14273
[19:22:53] [DEBUG]  FILE | File transfer complete!
[19:22:53] [DEBUG]   GEN | FD  Disconnect! Connection state 1 Should flush: 1
[19:22:59] [DEBUG]     1 | [Permission] Value test result for test type 1.
[19:22:59] [DEBUG]     1 | [Permission]    Permission: i_client_private_textmessage_power Required value: 60 Gained value: 60 Force required: 0 Channel: Max 2 #1 Result: 1
[19:24:28] [INFO]    GEN | [WebList] Sending new report.
[19:24:28] [INFO]    GEN | Weblist successfull updated
[19:24:28] [INFO]    GEN | [WebList] Report successfully send!
[19:30:30] [DEBUG]   GEN | Invalid packet id pushback. Current index: 39 packet index: 38 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0039 connection_server2client_packetloss_keepalive=0.0000 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0000)
[19:31:29] [DEBUG]   GEN | Invalid packet id pushback. Current index: 40 packet index: 39 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0039 connection_server2client_packetloss_keepalive=0.0000 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0000)
[19:34:28] [INFO]    GEN | [WebList] Sending new report.
[19:34:28] [INFO]    GEN | Weblist successfull updated
[19:34:28] [INFO]    GEN | [WebList] Report successfully send!
[19:34:30] [DEBUG]   GEN | Invalid packet id pushback. Current index: 48 packet index: 47 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0039 connection_server2client_packetloss_keepalive=0.0000 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0000)
[19:34:30] [DEBUG]   GEN | Invalid packet id pushback. Current index: 48 packet index: 47 (Data: connectioninfoautoupdate connection_server2client_packetloss_speech=0.0039 connection_server2client_packetloss_keepalive=0.0000 connection_server2client_packetloss_control=0.0000 connection_server2client_packetloss_total=0.0000)
[19:37:31] [ERROR]     1 | Found delay while ticking server. (Delay: 510ms)
[19:37:38] [DEBUG]     1 | [IP:45590/undefined] Creating VoiceClient instance at 0x7f155d1d5b80
[19:37:38] [DEBUG]     1 | Allocated new voice client connection at 0x7f155f5a32d0
[19:37:38] [DEBUG]     1 | Remote address (v4) IP destination address IP
[19:37:38] [DEBUG]     1 | Having new voice client. Remote address: IP:45590
[19:37:38] [DEBUG]     1 | Failed handshake for IP (Invalid server cookie response!)
[19:37:38] [DEBUG]     1 | [IP:45590/undefined] Closing voice client connection. (Flush: false)
[19:37:38] [INFO]      1 | Voice client 0/ (undefined) from IP:45590 left.
[19:37:38] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_last_client_disconnect. New value: 1531589858. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:37:38] [DEBUG]     1 | [IP:45590/undefined] Creating VoiceClient instance at 0x7f1556fc8240
[19:37:38] [DEBUG]     1 | Allocated new voice client connection at 0x7f15574ff1d0
[19:37:38] [DEBUG]     1 | Remote address (v4) IP destination address IP
[19:37:38] [DEBUG]     1 | Having new voice client. Remote address: IP:45590
[19:37:38] [DEBUG]     1 | [IP:45590/undefined]Invalid handshake packet state 2. Current version: 0
[19:37:39] [DEBUG]     1 | [IP:45590/undefined]Invalid handshake packet state 2. Current version: 0
[19:37:40] [DEBUG]     1 | [IP:45590/undefined]Invalid handshake packet state 2. Current version: 0
[19:37:41] [DEBUG]     1 | [IP:45590/undefined]Invalid handshake packet state 2. Current version: 0
[19:37:47] [DEBUG]     1 | [IP:45590/undefined] Got handshake timeout. IP:45590. State: INIT_LOW Time: 5
[19:37:47] [DEBUG]     1 | [IP:45590/undefined] Closing voice client connection. (Flush: true)
[19:37:47] [DEBUG]     1 | [IP:45590/undefined] Awaiting write prepare, write and acknowledge queue flushed
[19:37:47] [DEBUG]     1 | [IP:45590/undefined] Write prepare queue progressed
[19:37:47] [DEBUG]     1 | [IP:45590/undefined] Write and acknowledge queue are flushed
[19:37:47] [INFO]      1 | Voice client 0/ (undefined) from IP:45590 left.
[19:37:47] [DEBUG]   GEN | [SERVER] Updating server property for server 1: virtualserver_last_client_disconnect. New value: 1531589867. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :sid AND `type` = :type AND `id` = :id AND `key` = :key
[19:37:50] [DEBUG]     1 | [IP:45590/undefined] Creating VoiceClient instance at 0x7f1556fc8240
[19:37:50] [DEBUG]     1 | Allocated new voice client connection at 0x7f155757f3a0
[19:37:50] [DEBUG]     1 | Remote address (v4) IP destination address IP
[19:37:50] [DEBUG]     1 | Having new voice client. Remote address: IP:45590
[19:37:50] [DEBUG]     1 | [IP:45590/undefined] Got initivexpand acknowlage!
[19:37:50] [DEBUG]     1 | Having new instance user on server 1. (Database ID: 96 Name: undefined)
```

**Comment by WolverinDEV** [comment]
Well, I cant see which parameter's missing.
You have to enable trace log or use TS3Hook to get the last packet which causes the disconnect.
`parameter not found` is a disconnect reason which fires client side

**Comment by daffyyyy** [comment]
>  You have to enable trace log or use TS3Hook to get the last packet which causes the disconnect.


Okay, I set the trace log. We will see in time whether something will happen

**Comment by xRedFox** [comment]
Hey @WolverinDEV i found something interesting and i have some questions <.<
Did you escaped correctly `{}` inside channel names? :)
Look at channel names with `{}`:
```
[21:04:32] [DEBUG]   169 | failed to format message '[CHANNEL] Updating channel property for channel 59 (Stanza Musica 2 {OFF}): channel_needed_talk_power. New value: '75'. Query: UPDATE `properties` SET `value` = :value WHERE `serverId` = :serverId AND `id` = :id AND `key` = :key AND `type` = :type': argument not found
```
```

[21:04:34] [DEBUG]   169 |  - 56/[spacer11]
[21:04:34] [DEBUG]   169 | failed to format message ' - 57/Stanza Musica 1 {off}
[21:04:34] [DEBUG] ': argument not found
[21:04:34] [DEBUG]   169 |  - 58/[spacer12]
[21:04:34] [DEBUG]   169 | failed to format message ' - 59/Stanza Musica 2 {OFF}
[21:04:34] [DEBUG] ': argument not found
[21:04:34] [DEBUG]   169 |  - 60/[spacer13]

```

**Comment by WolverinDEV** [comment]
Oh yeah that's just a bug within the log system because I used the ond one there.
I changed it some time ago but still not updated all messages. But it does not really matters

**Comment by WolverinDEV** [comment]
Hey,
does it still happen? Or does it disappeared my magic?

**Comment by daffyyyy** [comment]
When I update the server and increase io limit / threads, it's okay

**Comment by h1rule** [comment]
what IO LIMIT / threads did you set it to @daffyyyy ?

**Comment by WolverinDEV** [comment]
Should be fixed within version 1.2.10-beta!
